pjmedia/src/pjmedia/transport_srtp_sdes.c

```diff
@@ -621,11 +621,14 @@ static pj_status_t fill_local_crypto(pj_pool_t *pool,
             return PJMEDIA_SRTP_ESDPINCRYPTOTAG;
 
         if (as_offerer) {
-            loc_crypto[loc_tag-1] = tmp_crypto;
+            if (loc_tag <= *count)
```

Comment:
I think Coverity may mark this as unnecessary since it's already established above that `loc_tag` cannot be > than `*count*`.